src/base/net/geoipmanager.cpp

```diff
@@ -372,10 +372,10 @@ QString GeoIPManager::CountryName(const QString &countryISOCode)
         {u"TM"_qs, tr("Turkmenistan")},
         {u"TN"_qs, tr("Tunisia")},
         {u"TO"_qs, tr("Tonga")},
-        {u"TR"_qs, tr("Turkey")},
+        {u"TR"_qs, tr("Turkiye")},
```

Comment:
It is _Türkiye_ according to ISO.

Reply:
This is true and it can be changed if you like, but I noticed that most countries with accent marks in their names have them missing in this list.